x-pack/platform/plugins/shared/maps/public/lens/passive_map_lazy.tsx

```diff
@@ -10,7 +10,7 @@ import { dynamic } from '@kbn/shared-ux-utility';
 import type { Props } from './passive_map';
 
 const Component = dynamic(async () => {
-  const { PassiveMap } = await import('./passive_map');
+  const { PassiveMap } = await import('../react_embeddable/embeddable_module');
```

Comment:
nit, is this directory jumping import necessary? seems a little strange to import a module (`passive_map`) that exists in this directory but it re-exported from a parent module (`react_embeddable/embeddable_module`).

Reply:
OK. I'm not that concerned about it. We can leave as is. Thanks for the explanation.